.github/ISSUE_TEMPLATE/integration_request.yaml

```diff
@@ -0,0 +1,78 @@
+name: New Tool Integration
+description: Keptn should integrate with this tool. Let's make it happen...
+title: "[integration] "
+labels: ["integrations"]
+assignees:
+  - agardnerit
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Keptn is designed to work with any tool. Use this form to request a new integration for a tool you use.
+
+        Before you start!
+
+        - If your tool is already containerised or can run as a script (PowerShell, Shell or Python), look into the [job executor service](https://github.com/keptn-contrib/job-executor-service).
+        - If you only need to notify a tool of a Keptn event, use the [webhook service](https://github.com/keptn/keptn/tree/master/webhook-service) (comes preinstalled with Keptn).
```

Comment:
It is worth clarifying that the target tool/service should have HTTP endpoints or REST API